packages/matchbox/src/components/Banner/Banner.js

```diff
@@ -12,35 +12,40 @@ import { buttonReset } from '../../styles/helpers';
 import { buttonFrom } from '../Button';
 import { margin } from 'styled-system';
 
-function IconSection({ status }) {
+function IconSection({ status, size }) {
   const statusIcon = React.useMemo(() => {
     return status === 'default' ? statusIcons.info : statusIcons[status];
   }, [statusIcons, status]);
 
   const Icon = statusIcon.iconName;
 
+  const iconSize = size === 'large' ? '3rem' : '1rem';
+  const bgColor = size === 'large' ? statusIcon.bg : null;
+  const fillColor = size === 'large' ? statusIcon.fill : statusIcon.bg;
+  const iconMargin = size === 'large' ? '500' : '300';
+
   return (
     <Box
       position="relative"
       display="flex"
       flexShrink="0"
       alignItems="center"
       justifyContent="center"
-      width="3rem"
-      height="3rem"
-      mr="500"
+      width={iconSize}
+      height={iconSize}
+      mr={iconMargin}
     >
       <Box
         position="absolute"
         top="0"
         left="0"
-        width="3rem"
-        height="3rem"
+        width={iconSize}
+        height={iconSize}
         borderRadius="circle"
-        bg={statusIcon.bg}
+        bg={bgColor}
       />
-      <Box position="relative" color={statusIcon.fill}>
-        <Icon size={30} label={statusIcon.iconLabel} />
+      <Box position="relative" color={fillColor}>
```

Comment:
The icons aren't centered with one line of text in the banner, i don't see an easy way to fix that though. if you add `mt="2px"` for small banners here it looks okay I think